app/views/tag_groups/_form.html.erb

```diff
@@ -0,0 +1,26 @@
+<% modal ||= false %>
```

Comment:
It will not be passed in the actions new and edit.
https://github.com/Codeminer42/cm42-central/blob/bfcb5ae86fbcd0365a103a47b2ddcdce76113ba0/app/views/tag_groups/new.html.erb#L17